var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -154,16 +156,21 @@ def run_cxx_example_test(self):
 
         test_dir = join_path(self.test_suite.current_test_cache_dir, 'examples', 'apps')
 
-        if not os.path.exists(test_dir):
-            print('Skipping caliper test')
+        if not os.path.isfile(join_path(test_dir, 'cxx-example.cpp')):
+            tty.msg('Skipping caliper test: file does not exist')
             return
 
         exe = 'cxx-example'
 
+        if os.path.exists(self.prefix.lib):
+            lib_dir = self.prefix.lib
+        else:
+            lib_dir = self.prefix.lib64
+
         self.run_test(exe='gcc',
```

Comment:
Nothing to do with this PR, but why is it using gcc and not the compiler from the spec? And why gcc and not g++?